### DISCUSSION TITLE
Github Registry Size Limit

### DISCUSSION THREAD

**Discussion by alberk8** [community/community#119737] [Packages] [answered]
### Select Topic Area

Question

### Body

Hi,
I understand that the Personal and Organization Free Tier comes with 500MB storage limit.  I have the following questions.

1) Is the 500MB limit the total size for all packages or a single package?  
2) Where can I view the current storage usage?

Thank you.

**Comment by grandmas-favorite** [comment, marked as answer]
Hi @alberk8!

1. The limit is for all packages, per month
2. Here's our instructions on viewing your Packages usage:
    - [Viewing your GitHub Packages usage](https://docs.github.com/en/billing/managing-billing-for-github-packages/viewing-your-github-packages-usage)


Hope that helps!

**Reply by alberk8** [reply]
Thank you for the reply.
